### ISSUE TITLE
Getting a bunch of deprecation warning from paramiko

### ISSUE THREAD

**Issue by benlapETS** [MetPX/sarracenia#161]
Here is what I got from running the flow test on master:
```
      1 sr_poll_f62_01.log [ERROR] /local/ben/anaconda3/envs/sarra-dev37/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:39: CryptographyDeprecationWarning: encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.
      1 sr_poll_f62_01.log [ERROR]   m.add_string(self.Q_C.public_numbers().encode_point())
      1 sr_poll_f62_01.log [ERROR] /local/ben/anaconda3/envs/sarra-dev37/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:96: CryptographyDeprecationWarning: Support for unsafe construction of public numbers from encoded data will be removed in a future version. Please use EllipticCurvePublicKey.from_encoded_point
      1 sr_poll_f62_01.log [ERROR]   self.curve, Q_S_bytes
```
The same apply for **sr_sender_tsource2send_f50_01.log**, **sr_subscribe_cp_f61_01.log**, **sr_subscribe_q_f71_01.log** and **sr_subscribe_u_sftp_f60_01.log**.

**Comment by petersilva** [comment]
The offending code is within paramiko, which is the library we use for SSH support.  It's a matter of paramiko making the appropriate adjustments.  For now, I think we just wait for paramiko to fix the issue.

**Comment by benlapETS** [comment]
From what i saw on [Paramiko github](https://github.com/paramiko/paramiko/issues/1386) it is a matter of time before this warning disapears and it is only an interface deprecation, nothing to be worried about

**Comment by benlapETS** [comment]
This raised me a concern, this means that we will catch everything from our dependencies that is throw in stderr and log it as ERROR in our log, is that a problem ?

**Comment by cowlinator** [comment]
FYI, paramiko has released 2.5.0, which no longer uses deprecated cryptography calls.
https://github.com/paramiko/paramiko/releases/tag/2.5.0
